utils/python/make_nexus_output_pretty.py

```diff
@@ -22,8 +22,12 @@ def get_hemco_dates(time_file='HEMCO_sa_Time.rc'):
             end = dt.datetime.strptime(L, 'END:     %Y-%m-%d %H:00:00\n')
         if L.startswith('TS_EMIS'):
             ts_emis = float(L.split()[1].strip('\n'))
+    print(start, start_base)
```

Comment:
I think could remove the changes here and merge #18

Reply:
good lets do that

Reply:
It's been merged

Reply:
And I just merged develop here